messageCheck/messageVerifier.go

```diff
@@ -1,14 +1,19 @@
 package messagecheck
 
 import (
+	"encoding/json"
+
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
 	"github.com/ElrondNetwork/elrond-go-core/marshal"
-	"github.com/ElrondNetwork/elrond-go-p2p"
-	"github.com/ElrondNetwork/elrond-go-p2p/message"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	p2p "github.com/ElrondNetwork/elrond-go-p2p"
+	"github.com/ElrondNetwork/elrond-go-p2p/libp2p"
 	pubsub "github.com/ElrondNetwork/go-libp2p-pubsub"
 	pubsub_pb "github.com/ElrondNetwork/go-libp2p-pubsub/pb"
 )
 
+var log = logger.GetOrCreate("p2p/messagecheck")
```

Comment:
messageVerifier?

Reply:
package name is messagecheck, i can maybe rename the package name..